lumen/ai/coordinator.py

```diff
@@ -710,8 +711,11 @@ async def _compute_execution_graph(self, messages: list[Message], agents: dict[s
                 execution_graph, unmet_dependencies = await self._resolve_plan(plan, agents, messages)
                 if unmet_dependencies:
                     istep.stream(f"The plan didn't account for {unmet_dependencies!r}", replace=True)
+                    attempts += 1
                 else:
                     planned = True
+                if attempts > 5:
+                    raise ValueError(f"Could not find a suitable plan for {messages[-1]["content"]!r}")
```

Comment:
We should probably start making sure that we provide friendly error messages when things go wrong and develop a general "tone" for how the system interacts with the user. 

Does it speak in first person:

> I could not find a suitable plan to answer your question because I could not find a way to determine the correct <table | data | ...>. Please rephrase your question.

is it apologetic:

> Sorry, I couldn’t come up with a good plan to answer your question—I couldn’t figure out the right <table | data | ...>. Could you try rephrasing it for me?

Does it use passive voice:

> A suitable plan to answer your question could not be found because the correct <table | data | ...> could not be determined. Please rephrase your question.

Reply:
That's not specific to this error, just an observation.